frontend/src/components/form/base/__tests__/__snapshots__/EColorPicker.spec.js.snap

```diff
@@ -30,194 +30,128 @@ exports[`An EColorPicker looks like a color picker: pickerclosed 1`] = `
 `;
 
 exports[`An EColorPicker looks like a color picker: pickeropen 1`] = `
-<body
-  data-app="true"
->
-  <div
-    class="v-menu__content theme--light menuable__content__active "
-    role="menu"
-    style="max-height: auto; min-width: 290px; max-width: 290px; top: 12px; left: 0px; transform-origin: top left; z-index: 8; display: none;"
-  >
-     
-    <div
-      class="v-card v-sheet theme--light"
-    >
-      <div
-        class="v-color-picker v-sheet theme--light v-color-picker--flat theme--light"
-        style="max-width: 300px;"
-      >
-        <div
-          class="v-color-picker__canvas"
-          style="width: 300px; height: 150px;"
-        >
-          <canvas
-            height="150"
-            width="300"
-          />
-          <div
-            class="v-color-picker__canvas-dot"
-            style="width: 10px; height: 10px; transform: translate(295px, -5px);"
-          />
-        </div>
-        <div
-          class="v-color-picker__controls"
-        >
-          <div
-            class="v-color-picker__preview v-color-picker__preview--hide-alpha"
-          >
-            <div
-              class="v-color-picker__dot"
-            >
-              <div
-                style="background: rgb(255, 0, 0);"
-              />
-            </div>
-            <div
-              class="v-color-picker__sliders"
-            >
-              <div
-                class="v-input v-color-picker__hue v-input--hide-details theme--light v-input__slider v-color-picker__track"
-              >
-                <div
-                  class="v-input__control"
-                >
-                  <div
-                    class="v-input__slot"
-                  >
-                    <div
-                      class="v-slider v-slider--horizontal theme--light"
-                    >
-                      <input
-                        disabled="disabled"
-                        id="input-26"
```

Comment:
here you see the id, which would be removed if jest-serializer-vue-tjw was used.